src/components/StampOverlay/StampOverlay.js

```diff
@@ -166,11 +211,33 @@ class StampOverlay extends React.Component {
         style={{ left, top }}
         data-element="stampOverlay"
       >
-        <div className="default-stamp-container">
-          <div className="modal-body">
-            { imgs }
+        <Tabs id="rubberStampTab">
+          <div className="header">
+            <div className="tab-list">
+              <Tab dataElement="defaultRubberStampButton">
+                <Button label={StandardBusiness} />
+              </Tab>
+              <Tab dataElement="customRubberStampButton">
+                <Button label={CustomStamp} />
+              </Tab>
+            </div>
           </div>
-        </div>
+
+          <TabPanel dataElement="defaultRubberStamp">
```

Comment:
Can we follow the naming conventions for other `TabPanel`s and end the `dataElement` with `Panel`? For example, we can have `defaultRubberStampPanel` here.